### PR TITLE
Fix memory leak

### PR DESCRIPTION
**Pull request by IMS212**

### CODE REVIEW

File: src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java [hunk 1]
```diff
@@ -45,18 +50,28 @@
 	private static final String RENDER_WEATHER = "Lnet/minecraft/client/render/WorldRenderer;renderWeather(Lnet/minecraft/client/render/LightmapTextureManager;FDDD)V";
 	private static final String RENDER_WORLD_BORDER = "Lnet/minecraft/client/render/WorldRenderer;renderWorldBorder(Lnet/minecraft/client/render/Camera;)V";
 
+	@Unique
+	HorizonRenderer horizonRenderer = new HorizonRenderer();
```
Comment (coderbot16) [inline src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java:54]: This can probably be private?

**Review by coderbot16 — CHANGES_REQUESTED** [review]

File: src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java (hunk 1 above)
Comment (IMS212) [reply src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java:54]: Sure, didn't think of much reason to keep it private but again, not too familiar with accessing.

Comment (IMS212) [reply src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java]: Fixed.

Comment (coderbot16) [reply src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java]: Yeah, it's best to keep fields and similar things private unless they need to be public, that's part of encapsulation.

**Review by coderbot16 — APPROVED** [review]
Thanks for finding this fix! Hopefully this resolves the remaining performance collapse & memory leak issues on 1.17.

**Pull request merged**